### DISCUSSION TITLE
kubernetes pack - manage multiple clusters

### DISCUSSION THREAD

**Discussion by dannietjoh** [StackStorm/st2#5739] [Packs]
I'm testing a Stackstorm-HA setup to manage many kubernetes clusters but the kubernetes pack only allows configuration of a single endpoint. Is there a way to get this to work with a single setup? Or does every cluster require its own Stackstorm setup?

**Comment by arm4b** [comment]
Considering how https://github.com/stackStorm-exchange/stackstorm-kubernetes works with dynamically creating sensors, I would probably create a copy of a k8s pack with unique config for each cluster you'd like to manage to keep them independent.
